api/tests/app/test_camera.py

```diff
@@ -0,0 +1,645 @@
+import pytest
+import os
+import base64
+import copy
+import numpy
+
+from api.tests.utils.common_functions import get_section_from_config_file, get_config_file_json
+
+# The line below is absolutely necessary. Fixtures are passed as arguments to test functions. That is why IDE could
+# not recognized them.
+from api.tests.utils.fixtures_tests import config_rollback, camera_sample, rollback_screenshot_camera_folder, h_inverse_matrix, pts_destination, rollback_homography_matrix_folder
+
+"""
+TO DO: Test stuffs related with the parameter reboot_processor.
+"""
+
+
+def create_a_camera(client, camera):
+    return client.post('/cameras', json=camera)
+
+
+def create_2_cameras(client, camera_base, enable=False):
+    camera_1 = copy.deepcopy(camera_base)
+    camera_2 = copy.deepcopy(camera_base)
+    camera_1['id'] = '321'
+    camera_2['id'] = '123'
+    camera_1['live_feed_enabled'] = enable
+    camera_2['live_feed_enabled'] = enable
+    client.post('/cameras', json=camera_1)
+    client.post('/cameras', json=camera_2)
+    return camera_1, camera_2
+
+
+
+def get_all_cameras(config_sample_path, with_image=False):
+    config_file_json = get_config_file_json(config_sample_path, decamelize=True)
+
+    list_of_cameras = {
+        'cameras': [config_file_json[camera] for camera in config_file_json if camera.startswith('source__')]}
+
+    for camera in list_of_cameras['cameras']:
+        camera.update({'id': str(camera['id'])})
+
+        if with_image:
+            from api.routers.cameras import get_camera_default_image_string
+            image_string = get_camera_default_image_string(camera['id'])
+            # Do not forget to decode it here, to turn into string.
+            camera["image"] = image_string.decode("utf-8")
+        else:
+            camera["image"] = None
+
+    return list_of_cameras
+
+
+def get_camera_from_config_file(camera_id, config_sample_path):
+    list_of_cameras = get_all_cameras(config_sample_path)['cameras']
+    for camera in list_of_cameras:
+        if camera['id'] == camera_id:
+            return camera
+    return None
+
+
+# pytest -v api/tests/app/test_camera.py::TestClassListCameras
+class TestClassListCameras:
```

Comment:
Not sure about conventions here but I think the "Class" part is not necessary. What I know about tests is that they are normally called like `TestListCameras`

Reply:
https://docs.pytest.org/en/reorganize-docs/new-docs/user/naming_conventions.html

Reply:
Yes, not necessary at all, it was just a personal convention. Should I remove them from all classes?

Reply:
Yeah I'd remove the "Class" part

Reply:
Okey, changed!